EIPS/eip-725.md

```diff
@@ -1,9 +1,9 @@
 ---
 eip: 725
-title: General key-value store and execution standard
+title: General data key/value store and execution standard
 author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
 discussions-to: https://github.com/ethereum/EIPs/issues/725
-status: Draft
+status: Review
 type: Standards Track
 category: ERC
 requires: 165, 173
```

Comment:
```suggestion
category: ERC
created: 2017-10-02
requires: 165, 173
```

Header fields are out of order (and I can't grab the existing `created:` header in this suggestion :sweat_smile:)